mmpose/models/detectors/pose_lifter.py

```diff
@@ -29,6 +34,10 @@ def __init__(self,
         self.train_cfg = train_cfg
         self.test_cfg = test_cfg
 
+        # semi-supervised learning
+        self.semi = keypoint_head is not None and traj_head is not None and \
```

Comment:
I suggest using a single condition here, e.g. "self.semi==loss_semi is not None", and check other related arguments with assertions.
BTW, is traj_backbone necessary for semi-supervised learning? What would happen if self.semi==True and traj_backbone is None?

Reply:
How about sharing backbone with pose model if traj_backbone is None?

Reply:
Yes, I agree. Please clarify the behavior in the docstring.